src/botcmd.c

```diff
@@ -62,11 +62,13 @@ static char base64to[256] = {
 
 int base64_to_int(char *buf)
 {
-  int i = 0;
+  int i = 0, j;
 
   while (*buf) {
-    i = i << 6;
-    i += base64to[(int) *buf];
+    j = base64to[(uint8_t) *buf];
+    if (i > ((INT_MAX >> 6) - j)) /* if overflow return -1 */
```

Comment:
INT_MAX >> 6 smells like UD because it overflows int, no?, the old way was better if that's the case